hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemFlush.java

```diff
@@ -400,7 +400,7 @@ private void validate(InputStream stream, byte[] writeBuffer, boolean isEqual)
             writeBuffer,
             readBuffer);
       } else {
-        assertThat(
+          org.hamcrest.MatcherAssert.assertThat(
```

Comment:
Indentation here looks off, I think we want to keep the original.

Reply:
I will fix it.